lib/gssapi/mech/gss_import_name.c

```diff
@@ -230,6 +232,8 @@ gss_import_name(OM_uint32 *minor_status,
                                                name_type, output_name);
 	}
 
+	_gss_mg_log(10, "Importing name \"%*.s\"", (int)input_name_buffer->length,
```

Comment:
Watch out!  A name can be binary-like.

Reply:
(Names are allowed to have embedded NULs.

Reply:
Hmmm, what can we do here? base64 encode it? We must have a way to log these things for debugging and troubleshooting.